### DISCUSSION TITLE
Oracle modules

### DISCUSSION THREAD

**Discussion by RandCNX** [cosmos/community#17] [Chain Builders (Cosmos SDK)] [answered]
Hello, I would like to ask if there is a module that acts as an oracle or if someone has a guide on how to implement one.

Thank you.

**Comment by julienrbrt** [comment, marked as answer]
Hi, I think you could have a look at Band Protocol (https://github.com/bandprotocol/chain/tree/master/x/oracle) or Ojo (https://github.com/ojo-network/ojo/tree/main/x/oracle)

**Reply by RandCNX** [reply]
Ok, thank you, I will take a look
